<ui version="4.0" >
 <class>NetworkPage</class>
 <widget class="QWidget" name="NetworkPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="networkBox" >
     <property name="title" >
      <string>&amp;Networking</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" >
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QCheckBox" name="networkCheck" >
          <property name="text" >
           <string>Enable network</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="netAccelCheck" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Use Accelerated Drivers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="networkFrame" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_28" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22" >
           <item>
            <widget class="QPushButton" name="advancedButton" >
             <property name="text" >
              <string>Advanced Settings</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24" >
           <item>
            <widget class="QStackedWidget" name="networkStack" >
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <property name="currentIndex" >
              <number>0</number>
             </property>
             <widget class="QWidget" name="page" >
              <property name="geometry" >
               <rect>
                <x>3</x>
                <y>3</y>
                <width>498</width>
                <height>296</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_31" >
               <property name="leftMargin" >
                <number>9</number>
               </property>
               <property name="topMargin" >
                <number>9</number>
               </property>
               <property name="rightMargin" >
                <number>9</number>
               </property>
               <item>
                <widget class="QLabel" name="label_19" >
                 <property name="text" >
                  <string>Select Network Interface Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21" >
                 <item>
                  <widget class="QComboBox" name="comboBox" >
                   <property name="sizePolicy" >
                    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <item>
                    <property name="text" >
                     <string>User</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>Bridged</string>
                    </property>
                   </item>
                   <item>
                    <property name="text" >
                     <string>Routed</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_11" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType" >
                    <enum>QSizePolicy::Preferred</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>Set MAC Address Automatically</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_18" >
                 <item>
                  <widget class="QLineEdit" name="lineEdit" >
                   <property name="enabled" >
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_9" >
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0" >
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_7" >
                 <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_2" >
              <property name="geometry" >
               <rect>
                <x>3</x>
                <y>3</y>
                <width>490</width>
                <height>267</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_19" />
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Custom networking options:</string>
           </property>
           <property name="buddy" >
            <cstring>networkEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" >
           <item>
            <widget class="QLineEdit" name="networkEdit" />
           </item>
           <item>
            <spacer name="horizontalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
